apps/meteor/client/components/Omnichannel/hooks/useDepartmentsList.ts

```diff
@@ -14,9 +15,10 @@ type DepartmentsListOptions = {
 	excludeDepartmentId?: string;
 	enabled?: boolean;
 	showArchived?: boolean;
+	selectedDepartment?: string;
 };
 
-type DepartmentListItem = {
+export type DepartmentListItem = {
```

Comment:
I recommend moving this to a separate module to avoid the cyclic dependency with `normalizeDepartments`